Fix gradle-wrapper.jar LFS issue

Zeerooth opened the pull request:
Resolve #2 
Some users may experience problems if they don't have git-lfs installed (that is, the LFS pointer gets copied into the project instead of actual gradle-wrapper.jar and generating ```java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain``` later on)
What I propose is simply moving this file from LFS storage to git repo - it is 52KB large anyway so I don't see much benefit in having it stored in git LFS

francesca64 commented:
Thanks for tackling this! It seems like LFS is less common than I'd assumed.

The only other file in LFS is similarly tiny: https://github.com/BrainiumLLC/cargo-mobile/blob/master/templates/apps/bevy/assets/branding/icon.png

It'd probably be a good idea to disable LFS outright.

Zeerooth commented:
> Thanks for tackling this! It seems like LFS is less common than I'd assumed.
> 
> The only other file in LFS is similarly tiny: https://github.com/BrainiumLLC/cargo-mobile/blob/master/templates/apps/bevy/assets/branding/icon.png
> 
> It'd probably be a good idea to disable LFS outright.

Yeah, I personally think that LFS is a great tool when you have to work on files that are hundreds of megabytes in size and when you work on the repo directly (i.e it is not some kind of dependency), but in this project it is bound to cause more problems that serve any benefit.

Anyway, I moved every file that was stored in git LFS - it was just a few more .png files from templates - and I also removed .gitattributes so that new files won't be accidentally added.

francesca64 reviewed (approved):
Great work!
